### DISCUSSION TITLE
2.3.61 incompatible with 2.3.4??

### DISCUSSION THREAD

**Discussion by raichea** [Bodmer/TFT_eSPI#1079] [Q&A - General] [answered]
I started playing with this library (v2.3.4) and a TTGO T-Display in Nov/Dec last year and it worked fine. Another project has occupied my free time since then until today when I started looking at it again. I'm working in PlatformIO and it told me that v2.3.61 was available as an update but won't allow me to apply it as it says it is incompatible.

I've skimmed the commits since 2.3.4 and the release comments but I don't see anything that suggests any problems with upgrading. Could you shed any light on what's going on here, please? I'd prefer to work with the current version if possible.

**Comment by Bodmer** [comment]
I have no idea why PlatformIO would report this. Users have successfully used 2.3.60 with PlatformIO.

A guess is that the PlatformIO configuration you are using has changed. Note: the TFT_eSPI library json file declares comatibility with ESP32, ESP8266 and STM32 processors only.

**Comment by raichea** [comment, marked as answer]
Doh. Apologies - rookie user error. I'd specified the exact version in my platformio.ini. After I'd fixed that, PlatformIO (not surprisingly) allowed me to upgrade. Thanks for the reply - it forced me to consider it was something dumb I'd done.
